### ISSUE TITLE
Input in pager does not work (jqGrid  4.9.0-rc1)

### ISSUE THREAD

**Issue by petronellius** [free-jqgrid/jqGrid#92]
When I change page number in pager text input and press ENTER nothing happens.
To simulate problem I downloaded working example from http://www.ok-soft-gmbh.com/jqGrid/DataWithCustomTitle.htm and took js and css from 4.9.0-rc1.

**Comment by petronellius** [comment]
I changed code for pager as below and it's working for me

$pagerIn.children(".ui-pg-table").bind("keydown.jqGrid", function (e) {
						var $focused;
						if (e.which === 13 && !$(e.target).is("input.ui-widget-content.ui-pg-input")) {
							$focused = $pagerIn.find(":focus");
							if ($focused.length > 0) {
								$focused.trigger("click");
								return false;
							}
						}
					});

**Comment by OlegKi** [comment]
Thank you for the bug report! I made some changes to allows basis keyboard navigation in the pager and in the navigator bar, but I made small errors. The origin of the problem was `return false` inside of `keydown` which break processing of `keypress` event. I removed `return false;` and [committed](https://github.com/free-jqgrid/jqGrid/commit/5fcac71920965e86e212739bb46de9c5bf3b99d2) the changes. Thanks you again!
